src/components/shared/PubmedEntry.jsx

```diff
@@ -25,62 +23,92 @@ class PubmedEntry extends React.Component {
       doid: '',
     };
 
+    this.makePubmedObj = this.makePubmedObj.bind(this);
+    this.makeDoiObj = this.makeDoiObj.bind(this);
     this.getPubmedInfo = this.getPubmedInfo.bind(this);
+    this.getDisplayDate = this.getDisplayDate.bind(this);
   }
 
-  getPubmedInfo() {
-    const { pub } = this.props;
-    let linkUrl = '#';
-    let linkDisable = true;
-    const paperInfo = pub;
-    let info = {
-      id: paperInfo.uid,
-      title: paperInfo.title,
-      authors: paperInfo.authors,
-      journal: paperInfo.fulljournalname,
-      source: paperInfo.source,
-      pubdate: paperInfo.pubdate,
-      url: `https://www.ncbi.nlm.nih.gov/pubmed/${paperInfo.uid}/`,
-      doid: paperInfo.elocationid,
+  makePubmedObj(publication) {
```

Comment:
This exists in two places? And it's different?

Reply:
In one place, we're making an object to display and it needs more fields. The other place is when the user wants to download a list of all publications on an edge where we don't need to include as much detail. Also, I just hacked this together.